File: 217-contains-duplicate/217-contains-duplicate.cs
public class Solution {
    public bool ContainsDuplicate(int[] nums) {
        for (int i = 0; i < nums.Length - 1; i++)
        {
            for (int j = nums.Length - 1; j > i; j--)
            {
            if (nums[i] == nums[j])
                return true;
            }
        }
        return false;
    }
}

File: 217-contains-duplicate/217-contains-duplicate.js
/**
 * @param {number[]} nums
 * @return {boolean}
 */
var containsDuplicate = function(nums) {
    var set = new Set();
    
    for (let num of nums) {
        if (num in set) return true;
        set[num] = true;
    }
    
    return false;
};